libs/ui/src/display-line/display-line.component.html

```diff
@@ -1,4 +1,19 @@
-<dt [class.info-bar-label-looks]="isBasicDisplay" [class.components-label-looks]="!isBasicDisplay">
+<dt (mouseenter)="toggleHover()" (mouseleave)="toggleHover()" *ngIf="!editing" [class.info-bar-label-looks]="isBasicDisplay" [class.components-label-looks]="!isBasicDisplay">
```

Comment:
Czyli jak podczas wpisywania myszką wyjade to sie wyłączy? Tak nie moze byc. Jezeli zaczełem wpisywac to nie moze wyłączyć edycji bez zapytania czy na bank chce usunac, albo cos podobnego

Reply:
Jak najedziesz myszką to pokazuje się przycisk do edycji i jak przejdziesz do edycji to jedyną opcją żeby ją wyłączyć jest naciśnięcie przycisku "ptaszka"